package.json

```diff
@@ -35,6 +35,7 @@
   "dependencies": {
     "@openzeppelin/contracts": "^4.0.0",
     "@razor-network/merkle": "^0.6.0",
+    "hardhat-contract-sizer": "^2.0.3",
```

Comment:
Is this required anymore?

Reply:
Done https://github.com/razor-network/contracts/pull/121/commits/e61016b7ecbf64a54dbbd3db02332c2addcd06af